src/Command/Service/CreateCoreCommandArchCommandService.php

```diff
@@ -0,0 +1,460 @@
+<?php
+
+namespace Centreon\Command\Service;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Centreon\Command\Model\DtoTemplate\RequestDtoTemplate;
+use Centreon\Command\Model\UseCaseTemplate\CommandUseCaseTemplate;
+use Centreon\Command\Model\PresenterTemplate\PresenterTemplate;
+use Centreon\Command\Model\RepositoryTemplate\RepositoryTemplate;
+use Centreon\Command\Model\ControllerTemplate\CommandControllerTemplate;
+use Centreon\Command\Model\FactoryTemplate\FactoryTemplate;
+use Centreon\Command\Model\ModelTemplate\ModelTemplate;
+use Centreon\Command\Model\PresenterTemplate\PresenterInterfaceTemplate;
+
+class CreateCoreCommandArchCommandService extends CreateCoreArchCommandService
```

Comment:
I would rather use composition over inheritance here for the services.

Reply:
Done